src/Context.ts

```diff
@@ -59,6 +46,10 @@ export class Context {
     return this.req.span;
   }
 
+  get meter(): opentelemetry.Meter {
+    return this.req.meter;
+  }
```

Comment:
Do we need this one now?

Reply:
Might not need it. I'll test locally if I can remove it completely

Reply:
Remove that and other meters